features/built_in_matchers/include.feature

```diff
@@ -1,7 +1,8 @@
 Feature: `include` matcher
 
-  Use the `include` matcher to specify that a collection includes one or more expected objects.
-  This works on any object that responds to `#include?` (such as a string or array):
+  Use the `include` matcher to specify that a collection includes one or more
+  expected objects. You can also think of it as an `any`-matcher. This works on
+  any object that responds to `#include?` (such as a string or array):
```

Comment:
> how i feel about thinking of include as an any-matcher, 

Mmmh... My thinking was that of a user. I saw that @myronmarston added the `all`-matcher sometime ago, but asked myself where the heck I find the `any`-matcher. I was not aware that the semantics of the `include`-matcher are the same until @myronmarston told me so. This is there reason why I added this stuff here.